.github/workflows/main.yml

```diff
@@ -15,15 +15,16 @@ jobs:
 
     strategy:
       matrix:
-        otp_version: ['23.2.1', '22.3.4.2']
+        otp_version: ['24', '23', '22']
```

Comment:
I think this achieves the same effect as you wanted before, but we don't need to be so specific with `setup-beam`.